pom.xml

```diff
@@ -105,6 +105,7 @@
         <maven.scm.version>1.11.2</maven.scm.version>
         <nexus-staging-plugin-version>1.6.8</nexus-staging-plugin-version>
         <wagon-ssh.version>3.3.2</wagon-ssh.version>
+        <jacoco-maven-plugin.version>0.8.6</jacoco-maven-plugin.version>
```

Comment:
Please alphabetize this between `exec-maven-plugin.version` and `maven-antrun-plugin.version`.

Reply:
done